static/js/chart/base.ts

```diff
@@ -271,9 +271,12 @@ function formatYAxisTicks(
   }
   const dollar = unit === "$" ? "$" : "";
   const percent = unit === "%" ? "%" : "";
+  const tons = unit === "t" ? "t" : "";
```

Comment:
should we also add these units to the chart config?

Reply:
Yep, added them back! I removed them due to the kkg issue that is now addressed by adding a space.